ambuda/templates/base.html

```diff
@@ -7,7 +7,7 @@
     A breakthrough Sanskrit library. Read our library of traditional Sanskrit texts with word-by-word analysis, integrated dictionary support, and so much more.
     {%- endblock %}">
     <link rel="stylesheet" type="text/css" href="/static/gen/style.css?v=10">
-    <link rel="icon" href="data:,">
+    <!-- <link rel="icon" href="data:,"> -->
```

Comment:
If this is absent, the page will (fail to) fetch a favicon on most requests, which is an extra hassle for slow connections. The proper long-term fix is to have a real favicon, but I see this as a workable stopgap.

Reply:
Re-added favicon and updated CSP appropriately.